<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Generate_dialog</class>
 <widget class="QDialog" name="Generate_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Generating new password</string>
  </property>
  <widget class="QLabel" name="paassword_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 11pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="reroll_button">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>206</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    background-color: rgb(0, 170, 127);
	font: 600 9pt &quot;Segoe UI Semibold&quot;;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgb(224, 0, 0);
    border-style: inset;
}</string>
   </property>
   <property name="text">
    <string>Reroll</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="no_special_checkbox">
      <property name="styleSheet">
       <string notr="true">QGroupBox {
	font: 700 11pt &quot;Segoe UI&quot;;
    alternate-background-color: rgb(0, 170, 127);
  border: 5px solid rgb(0, 170, 127);;
  padding: 10px;
  border-radius: 30px 20px;
}
</string>
      </property>
      <property name="text">
       <string>No special characters</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="addPasswordButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>250</y>
     <width>206</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    background-color: rgb(0, 170, 127);
	font: 600 9pt &quot;Segoe UI Semibold&quot;;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    font: bold 14px;
    min-width: 10em;
    padding: 6px;
}
QPushButton:pressed {
    background-color: rgb(224, 0, 0);
    border-style: inset;
}</string>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	font: 700 11pt &quot;Segoe UI&quot;;
    alternate-background-color: rgb(0, 170, 127);
  border: 3px solid rgb(0, 170, 127);;
  padding: 10px;
  border-radius: 30px 20px;
}
</string>
   </property>
   <property name="title">
    <string>Length:</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QRadioButton" name="short_checkbox_2">
      <property name="text">
       <string>8-12</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="middle_checkbox_2">
      <property name="text">
       <string>13-19</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="long_checbox_2">
      <property name="text">
       <string>20-31</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
